index.ipynb

```diff
@@ -0,0 +1,331 @@
+{
```

Comment:
<p>Still having trouble seeing the results with quick background after this sentence: we see degradation of Zernike estimate departing further from OPD as the source brightness decreases (source magnitude increases)</p><p>Can you make them more like the noBkgrd results above them?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/sitcomtn-085/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>Added plots for <code>qckBkgnd</code> that only include two magnitudes  - brightest (12) and faintest (17) to more cleanly show the difference from the OPD,  and that the main difference is in the source magnitude used, rather than whether we're performing averaging vs stacking.  </p>